### ISSUE TITLE
Possible do_not_annotate changes

### ISSUE THREAD

**Issue by kimrutherford** [pombase/pombase-config#100]
We should keep an eye on this issue:
   - geneontology/go-site#2107

because we might need to update our configuration if there are GO changes.

**Comment by ValWood** [comment]
Yes it isn't so urgent becasuse we rarely use the do not annotate term (usually we suggest them), and also I would pick them up via the GO logs.

We need to decide on the best messaging because sometimes people will want to 'park' at a 'do not annotate' term and suggest a new term, so we might not want to prevent it.
So far, I don't think any of the community have ever used a 'do not annotate'

**Comment by kimrutherford** [comment]
> So far, I don't think any of the community have ever used a 'do not annotate'

That's not possible in Canto due to the do_not_annotate flags in GO.   If the names or meanings change we'll need to change the Canto config.

**Comment by ValWood** [comment]
I think it's proba bly OK, it is all terms they shouldn't be using.
I'll keep an eye on it.
